vis_poly/cosmo_poly.py

```diff
@@ -0,0 +1,112 @@
+"""vizualise solvent accessible surface in .cosmo data"""
```

Comment:
Can you add contact information at the top here?

Reply:
added.